res/layout/fragment_media_player.xml

```diff
@@ -109,11 +109,18 @@
                 android:text="Playlist" />
 
             <CheckBox
-                android:id="@+id/loopingButton"
+                    android:id="@+id/subtitlesButton"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:paddingRight="5dp"
+                    android:text="Subtitles" />
+
+            <CheckBox
+                android:id="@+id/loopButton"
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:paddingRight="5dp"
-                android:text="Looping" />
+                android:text="Loop" />
```

Comment:
Please please please separate different changes in different commits (or even different PRs). It would be easier to browse history and revert specific functionality if required. Looping is not related to "Add subtitles checkbox".